# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1311694439 -7200
#      Tue Jul 26 17:33:59 2011 +0200
# Node ID 98bde3b5b7a05a61970cef5ca6453ec393527388
# Parent  e02a2fb08bfd492d93f6959f4139eae7819bab29
File browser does not show all information.

diff --git a/gui/src/FilesDockWidget.cpp b/gui/src/FilesDockWidget.cpp
--- a/gui/src/FilesDockWidget.cpp
+++ b/gui/src/FilesDockWidget.cpp
@@ -63,16 +63,19 @@ FilesDockWidget::FilesDockWidget (QWidge
 
   // Attach the model to the QTreeView and set the root index
   m_fileTreeView = new QTreeView (widget ());
   m_fileTreeView->setModel (m_fileSystemModel);
   m_fileTreeView->setRootIndex (rootPathIndex);
   m_fileTreeView->setSortingEnabled (true);
   m_fileTreeView->setAlternatingRowColors (true);
   m_fileTreeView->setAnimated (true);
+  m_fileTreeView->setColumnHidden (1, true);
+  m_fileTreeView->setColumnHidden (2, true);
+  m_fileTreeView->setColumnHidden (3, true);
   setCurrentDirectory (m_fileSystemModel->fileInfo (rootPathIndex).
 		       absoluteFilePath ());
 
   connect (m_fileTreeView, SIGNAL (doubleClicked (const QModelIndex &)), this,
 	   SLOT (itemDoubleClicked (const QModelIndex &)));
 
   // Layout the widgets vertically with the toolbar on top
   QVBoxLayout *
